libfreerdp/primitives/test/TestPrimitivesSet.c

```diff
@@ -62,7 +61,7 @@ static BOOL test_set8u_func(void)
 	for (off = 0; off < 16; ++off)
 	{
 		UINT32 len;
-		memset(dest, 3, sizeof(dest));
+		BYTE dest[1024] = { 3 };
```

Comment:
Same here. You change the test here. `{ 3 }` sets the first element to `3` and the rest (which is not specified) is automatically set to `0`, while before everything was set to `3`.

Reply:
reverted, good catch